pkg/arvo/sys/vane/ames.hoon

```diff
@@ -2004,86 +2096,82 @@
     |=  =ship
     ^+  event-core
     =+  (trace msg.veb ship |.("requesting attestion"))
-    =.  event-core  (send-blob | ship (sendkeys-packet ship))
+    =.  event-core  (send-packet | ~ (sendkeys-packet ship))
     =/  =wire  /alien/(scot %p ship)
     (emit duct %pass wire %b %wait (add now ~s30))
-  ::  +send-blob: fire packet at .ship and maybe sponsors
-  ::
-  ::    Send to .ship and sponsors until we find a direct lane,
-  ::    skipping .our in the sponsorship chain.
+  ::  +send-packet: fire packet at lane and maybe next
   ::
   ::    If we have no PKI data for a recipient, enqueue the packet and
   ::    request the information from Jael if we haven't already.
   ::
-  ++  send-blob
-    ~/  %send-blob
-    |=  [for=? =ship =blob]
+  ++  send-packet
+    |=  [for=? next-lane=(unit lane) =packet]
     ::
-    =/  final-ship  ship
-    %-  (trace rot.veb final-ship |.("send-blob: to {<ship>}"))
-    |-
-    |^  ^+  event-core
-        ::
-        =/  ship-state  (~(get by peers.ames-state) ship)
-        ::
-        ?.  ?=([~ %known *] ship-state)
-          ?:  ?=(%pawn (clan:title ship))
-            (try-next-sponsor (^sein:title ship))
-          %+  enqueue-alien-todo  ship
-          |=  todos=alien-agenda
-          todos(packets (~(put in packets.todos) blob))
-        ::
-        =/  =peer-state  +.u.ship-state
-        ::
-        ::  XX  routing hack to mimic old ames.
-        ::
-        ::    Before removing this, consider: moons when their planet is
-        ::    behind a NAT; a planet receiving initial acknowledgment
-        ::    from a star; a planet talking to another planet under
-        ::    another galaxy.
-        ::
-        ?:  ?|  =(our ship)
-                ?&  !=(final-ship ship)
-                    !=(%czar (clan:title ship))
-                ==
+    =/  final-ship  rcvr.packet
+    %-  (trace rot.veb final-ship |.("send-packet: to {<ship>}"))
```

Comment:
Should this be `{<final-ship>}` as `ship` is no longer an alias of `final-ship`?

Reply:
yes I'll change that